src/main/java/uk/ac/cam/cl/dtg/isaac/api/PagesFacade.java

```diff
@@ -381,6 +382,21 @@ public final Response getQuestionList(@Context final Request request,
             }
         }
 
+        try {
+            if (Objects.equals(statuses, "")) {
```

Comment:
If statuses is `null`, this won't catch it and it will cause an unhandled NPE in the else when it tries to split it. 
```suggestion
            if (null == statuses || Objects.equals(statuses, "")) {
```